routes/story.js

```diff
@@ -1,39 +1,61 @@
 const express = require("express");
 const router = express.Router();
+const Story = require('../models/Story');
 
 // @desc Story summary
 // @route GET /story/:storyId
 // @access Public
 router.get("/:storyId", (req, res) => {
```

Comment:
```suggestion
router.get("/:storyId", async (req, res) => {
```
Try making this an async function so you can use await to get stories data.